### ISSUE TITLE
Various left-right arrow key glitches

### ISSUE THREAD

**Issue by lairdshaw** [martec/automention#22]
Moving left or right with arrow keys in incomplete mentions results in various glitches, from skipping to the beginning of the mention, to the pop-up failing to pop up again when arrowing to the end of an incomplete mention, and even to auto-completed mentions that cannot be deleted. I'm not providing recipes for reproduction as these issues appear readily via a bit of playing around with the left-right arrow keys, but if others can't reproduce them, then I will provide one or two recipes.

I have spent a lot of time trying to identify where in the `atwho.js` code these issues arise and why, with no success. It is foreign (to me) code, and I just don't have the patience to pore over it enough to understand it sufficiently to be capable of fixing these issues. I continue to be "bugged by this bug".

**Comment by martec** [comment]
@lairdshaw 

first of all thanks for the PR. Merged!

about the problem pointed out, thank you very much for your time and dedication.
It would be better if these problems did not exist. Anyway this plugin has existed for more than 5 years with these problems and without much complaint. So I think we can live with that.

**Comment by lairdshaw** [comment]
Yes, I think we can live with it. My guess is that most members aren't fiddling around with left and right arrows on partial mentions anyway.

Thanks for merging the PRs and for freely providing this plugin. On its next stable release, I am going to install it and DVZ Mentions on Psience Quest. Together, they make a very compelling case to an admin, and I think our members are going to love them!
